config/initializers/assets_manifest.rb

```diff
@@ -0,0 +1,4 @@
+Rails.application.config.assets_manifest =
+  if File.exist?(Rails.root.join('public', 'assets', 'build', 'manifest.json'))
+    JSON.parse(File.read(Rails.root.join('public', 'assets', 'build', 'manifest.json')))
```

Comment:
__[RuboCop]__ <sub>posted by Sider</sub>

Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.